### PR TITLE
Liuda Leas task 8 (optional)

### PR DESCRIPTION
**Pull request by LiudaL**
Optional tasks added

### CODE REVIEW

File: src/main/java/Leas_Liudmila/files/GradeBook.java [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+
+    private String convertStudentToCsvLine(Student student) {
+        String studentInCsv = "";
+
+        studentInCsv += student.getId() + ",";
+        studentInCsv += student.getFirstName() + ",";
+        studentInCsv += student.getLastName() + ",";
+
+        for (int grade : student.getGrades()) {
+            studentInCsv += grade + ",";
```
Comment (dgrimaliuc-crunchyroll) [inline src/main/java/Leas_Liudmila/files/GradeBook.java:100]: in this case all grades will be in a different cell in csv file, but they should be in the same cell

File: src/main/java/Leas_Liudmila/files/GradeBook.java [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+        }
+
+        ArrayList<Student> result = new ArrayList<Student>();
+
+        Scanner read = new Scanner(path);
+
+        while (read.hasNextLine()) {
+            String csvStudent = read.nextLine();
+
+            if(csvStudent.length() > 1) {
```
Comment (dgrimaliuc-crunchyroll) [inline src/main/java/Leas_Liudmila/files/GradeBook.java:52]: in order to skip first line with column names you can before loop make read.nextLine() once

**Review by dgrimaliuc-crunchyroll — APPROVED** [review]
Cool

File: src/main/java/Leas_Liudmila/files/GradeBook.java (hunk 2 above)
Comment (LiudaL) [reply src/main/java/Leas_Liudmila/files/GradeBook.java:52]: This verification skips empty lines, not column names

**Pull request merged**